framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -305,5 +305,36 @@ int cfg_get_hostap_iface(int32_t radio_num, char hostap_iface[BPL_IFNAME_LEN])
     return cfg_get_prplmesh_radio_param(radio_num, "hostap_iface", hostap_iface, BPL_IFNAME_LEN);
 }
 
+int cfg_get_all_prplmesh_wifi_interfaces(BPL_WLAN_IFACE *interfaces, uint8_t *num_of_interfaces)
+{
+    if (!interfaces) {
+        MAPF_ERR("cfg_get_all_prplmesh_wifi_interfaces: invalid input: interfaces is NULL");
+        return RETURN_ERR;
+    }
+    if (!num_of_interfaces) {
+        MAPF_ERR("cfg_get_all_prplmesh_wifi_interfaces: invalid input: num_of_interfaces is NULL");
+        return RETURN_ERR;
+    }
+    if (*num_of_interfaces < 1) {
```

Comment:
I prefer to compare to 0 (nitpick)

Reply:
in response to a previous comment, the type was changed to int, thus this condition remains as is.